tests/test.sh

```diff
@@ -826,9 +856,10 @@ run_ice "$testdir/includes.o" "remote" 0 $GXX -Wall -Werror -c includes.cpp -o "
 run_ice "$testdir/plain.o" "local" 0 $GXX -Wall -Werror -c plain.cpp -mtune=native -o "$testdir"/plain.o
 run_ice "$testdir/plain.o" "remote" 0 $GCC -Wall -Werror -x c++ -c plain -o "$testdir"/plain.o
 if test -z "$debug_fission_disabled"; then
-    run_ice "" "remote" 1 "split_dwarf" $GXX -gsplit-dwarf -c nonexistent.cpp
+    run_ice "" "remote" "1 4" "split_dwarf" $GXX -gsplit-dwarf -c nonexistent.cpp
```

Comment:
g++ (Ubuntu 4.8.4-2ubuntu1~14.04.3) 4.8.4 returns 4 when the input file does not exist.